docs/api/use-form.md

```diff
@@ -240,6 +241,7 @@ This is your form submission handler, witch will pass your form's `values`. But
   | Name          | Description                   |
   |---------------|-------------------------------|
   |`setSubmitting`| Set isSubmitting imperatively.|
+  |`initialValue` | Form Initial Value. Update when ResetForm.           |
```

Comment:
```suggestion
  |`initialValue` | Form initial values.           |
```

How about thisHow about writing it like this?

Reply:
OK